### PR TITLE
GetCharge exported to cache

### PR DESCRIPTION
**Pull request by vgogte**
I have exported GetCharge member function to cache.h. GetCharge returns charge of the cache handle.

### CODE REVIEW

**Comment by siying** [comment]
Can you add a simple unit test for it?

**Comment by vgogte** [comment]
Hi Siying, I have added a unit test that checks for GetCharge in my branch.

File: cache/cache_test.cc [hunk 1]
```diff
@@ -686,6 +686,13 @@ TEST_P(CacheTest, DefaultShardBits) {
   ASSERT_EQ(6, sc->GetNumShardBits());
 }
 
+TEST_P(CacheTest, GetCharge) {
+  Insert(1, 2);
+  Cache::Handle* h1 = cache_->Lookup(EncodeKey(1));
+  ASSERT_EQ(2, DecodeValue(cache_->Value(h1)));
+  ASSERT_GT(cache_->GetCharge(h1), 0);
```
Comment (siying) [inline cache/cache_test.cc]: Why not  `ASSERT_EQ(1, ache_->GetCharge(h1));`?

Comment (vgogte) [reply cache/cache_test.cc]: Yes, it can be that. I have updated the commit.

**Pull request closed without merging**